src/bridge/transactions/burn.rs

```diff
@@ -77,11 +77,10 @@ impl BurnTransaction {
     pub fn pre_sign(&mut self, context: &VerifierContext) { self.sign_input0(context); }
 
     pub fn add_output(&mut self, output_script_pubkey: ScriptBuf) {
-        let output_index = 1;
-        self.tx.output[output_index] = TxOut {
+        self.tx.output.push(TxOut {
```

Comment:
I wonder if we should allow pushing multiple values here. Do we need to allow multiple additional outputs?

If we always use the same output index, users can override their values but not append an arbitrary number of additional outputs.

Reply:
Now we allow adding only one output. The output amount is predefined, user must only add the script.